framework/decode/file_processor.h

```diff
@@ -158,7 +159,6 @@ class FileProcessor
     std::vector<uint8_t>                parameter_buffer_;
     std::vector<uint8_t>                compressed_parameter_buffer_;
     util::Compressor*                   compressor_;
-    uint64_t                            api_call_index_;
```

Comment:
Incidental changes like this which do not affect the PR are welcome to come in as separate PRs that we can approve fast. If possible it is good to at least have them in their own commits so the PR is atomic when it lands on our dev branch.

Reply:
This change has been reverted. Will be fixed with another PR.

Reply:
Great, I wasn't expecting that for this PR, but thank you!